app/src/main/res/layout/item_skill.xml

```diff
@@ -9,13 +9,14 @@
     <android.support.v7.widget.CardView
         android:layout_width="@dimen/websearch_card_width"
         android:layout_height="wrap_content"
-        android:layout_marginBottom="@dimen/margin_small"
-        android:layout_marginLeft="@dimen/margin_small"
         android:layout_marginStart="@dimen/margin_small"
+        android:layout_marginLeft="@dimen/margin_small"
```

Comment:
Unnecessary change, revert

Reply:
sorry, my bad... that was inaccuracy

Reply:
@pallav12 here I meant to say that the values margin_left and margin_right have not changed, but it is unnecessary. change in position of these attributes. Please correct all these mistakes in the entire file.

Reply:
Okay... Now there's no such change which add +1 in "difference" just because reordering of attributes. Just squashed all commits and force pushed it.